client/app/telecom/pack/migration/building-details/telecom-pack-migration-building-details.html

```diff
@@ -0,0 +1,69 @@
+<div class="telecom-pack-migration-shipping">
+    <div class="row">
+        <div class="col-xs-12 col-md-6">
+            <div class="widget-presentation">
+                <!-- LOADER -->
+                <div class="text-center" data-ng-if="$ctrl.loading.init">
+                    <oui-spinner></oui-spinner>
+                </div>
+
+                <div data-ng-if="!$ctrl.loading.init" class="container">
+                    <div>
```

Comment:
This `div` could be removed if for instance you apply a `d-block` class directly on the `<span>`.